README.md

```diff
@@ -103,4 +104,43 @@ input = wrapField(Input);
   component={this.input}
   className={styles.input}
 />
-```
\ No newline at end of file
+```
+
+### Spinner
+
+The component contain this props:
+* WrappedComponent
+* classNameContainer
+* classNameLoading
+* idLoading
+* colorSpinner
+
+Contain this component:
+* Loading 
+  * className
+  * id
+  * colorSpinner
+
+
+In the example we will use it with the SearchBar component.
+
+```
+src/components/SearchBar
+
+//Import into the component to be used.
+import { withSpinner } from '../Spinner';
+
+//Include in the definition of the component to be used. In this case is SearchBar.
+export default withSpinner({
+  WrappedComponent: SearchBarContainer,
+  idLoading: 1,
+  classNameContainer: styles.spinner
+});
+
+```
+
+Here you can see all the spinner models that this library has http://kyleamathews.github.io/react-spinkit/ 
```

Comment:
are we using the whole number of models for this spinners? I'd be nice to have a demo of each of them and the corresponding loading id